frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -88,8 +87,8 @@ export function getField(activity, fieldName, defaultValue, prefix=''){
  * 
  * @param {string} tripId Database ID of the trip whose actiivty should be modified.
  * @param {string} activityId Database ID of the activity to be modified.
- * @param {Object} newValues Dictionary of the new values in {fieldName: newValue} form
- * @returns {boolean} true if the write was successful, false otherwise.
+ * @param {Object} newValues Dictionary of the new values in <code>{fieldName: newValue}</code> form
+ * @returns {boolean} <code>true</code> if the write was successful, <code>false</code> otherwise. 
```

Comment:
Update to @return and not @returns. Remove extra space at the end of the line as well.